src/ssh/ssh_session.cpp

```diff
@@ -40,6 +42,8 @@ mp::SSHSession::SSHSession(const std::string& host, int port, const std::string&
 
     const long timeout_secs = std::chrono::duration_cast<std::chrono::seconds>(timeout).count();
     const int nodelay{1};
+    auto ssh_dir =
+        QString("%1/.ssh").arg(QStandardPaths::writableLocation(QStandardPaths::AppConfigLocation)).toStdString();
```

Comment:
Two requests on the `QString("%1/.ssh")` bit (if it is to support an actual directory):
  - could this use `QDir::filePath` or a similar way to abstract the path for all platforms
  - could the directory not start with dot, just to play nicer on windows?

Reply:
You're right, it would be better to use `QDir` for that.  And yeah, really no need for the `.`, so I'll remove that.

Reply:
Ok, I pushed the changes requested here.